### DISCUSSION TITLE
[General] Account Abstraction Transaction Lifecycle - "The System" vs "The validator" vs "The node" vs "The sequencer"

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [zkSync-Community-Hub/zksync-developers#565] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

I'm looking to clarify something about the [transaction lifecycle.](https://docs.zksync.io/build/developer-reference/account-abstraction.html#the-transaction-flow) of an account abstraction transaction, but probably also just all transactions.

When a transaction is sent to an RPC endpoint (any transaction, type 113 or other) who receives it? The sequencer is also all the validators as of today, so the sequencer/validator gets it? The documentation just calls it `The system`. Is `The system` a validator? The sequencer? Something else?

**Comment by bxpana** [comment, marked as answer]
> When a transaction is sent to an RPC endpoint (any transaction, type 113 or other) who receives it? The sequencer is also all the validators as of today, so the sequencer/validator gets it?

Yes that is correct. 

> The documentation just calls it `The system`. Is `The system` a validator? The sequencer? Something else?

`The system` is referring to the infra of the system overall. This is something that we could be more clear about and will work on in the docs.
